pkg/controllers/dashboard/hostedkubernetecharts/controller.go

```diff
@@ -0,0 +1,106 @@
+package hostedkubernetecharts
+
+import (
+	"context"
+	"os"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/catalogv2/system"
+	"github.com/rancher/rancher/pkg/settings"
+	"github.com/rancher/rancher/pkg/wrangler"
+)
+
+var (
+	AksCrdChart = chartDef{
+		ReleaseNamespace: "cattle-system",
+		ChartName:        "rancher-aks-operator-crd",
+	}
+	AksChart = chartDef{
+		ReleaseNamespace: "cattle-system",
+		ChartName:        "rancher-aks-operator",
+	}
+	EksCrdChart = chartDef{
+		ReleaseNamespace: "cattle-system",
+		ChartName:        "rancher-eks-operator-crd",
+	}
+	EksChart = chartDef{
+		ReleaseNamespace: "cattle-system",
+		ChartName:        "rancher-eks-operator",
+	}
+	GkeCrdChart = chartDef{
+		ReleaseNamespace: "cattle-system",
+		ChartName:        "rancher-gke-operator-crd",
+	}
+	GkeChart = chartDef{
+		ReleaseNamespace: "cattle-system",
+		ChartName:        "rancher-gke-operator",
+	}
+)
+
+type chartDef struct {
+	ReleaseNamespace string
+	ChartName        string
+}
```

Comment:
I wonder if we should standardize and/or centralize charts references across our controllers.

- https://github.com/rancher/rancher/blob/a56d10297a277dcc04aa48c6dfd45022dda859e3/pkg/controllers/dashboard/systemcharts/controller.go#L17-L50
- https://github.com/rancher/rancher/blob/a56d10297a277dcc04aa48c6dfd45022dda859e3/pkg/controllers/dashboard/fleetcharts/controller.go#L15-L33

Reply:
That's a good one, I am going to standardize it.